spec/acceptance/linux_spec.rb

```diff
@@ -0,0 +1,21 @@
+# run a test task
+require 'spec_helper_acceptance'
+# bolt regexes
+# expect_multiple_regexes(result: result, regexes: [%r{"status":"(stopped|in_sync)"}, %r{Ran on 1 node}])
+# expect_multiple_regexes(result: result, regexes: [%r{"status":"stopped"}, %r{"enabled":"false"}, %r{Ran on 1 node}])
+
+describe 'linux service task', unless: fact_on(default, 'osfamily') == 'windows' do
+  package_to_use = ''
+  before(:all) do
+    if fact_on(default, 'osfamily') == 'RedHat' && fact_on(default, 'operatingsystemrelease') < '6'
+      run_task(task_name: 'service::linux', params: 'action=stop name=syslog')
+    end
+    package_to_use = 'rsyslog'
+    apply_manifest("package { \"#{package_to_use}\": ensure => present, }")
+  end
+  describe 'stop action' do
+    it 'stop/status a service' do
+      expect_multiple_regexes(result: run_task(task_name: 'service::linux', params: "action=stop name=#{package_to_use}"), regexes: [%r{status.*(stop)}, %r{#{task_summary_line}}])
```

Comment:
pulling the `run_task()` into a separate line would make this much more readable...